src/api/sharedPageAPI.js

```diff
@@ -0,0 +1,32 @@
+const BASEURL = 'https://bootcamp-api.codeit.kr/api';
+
+export const userDataAPI = async () => {
+  const APIData = { email: null, image: null };
+  const response = await fetch(`${BASEURL}/sample/user`);
+  if (response.ok) {
+    const result = await response.json();
+    APIData.email = result.email;
+    APIData.image = result.profileImageSource;
+    return APIData;
+  }
+  return null;
```

Comment:
```suggestion
  throw new Error('response not ok')
```

api에서는 null을 반환하기 보다는 에러인 상황에 대해 명시적으로 에러를 throw 해주는게 컴파일, 런타임 단계에서 모두 유리하게 작용합니다.